Assets/MRTK/Tests/PlayModeTests/SolverTests.cs

```diff
@@ -1099,13 +1099,7 @@ public IEnumerator TestFollowOrientation()
 
             Assert.AreEqual(targetTransform.rotation, Quaternion.identity, "Target rotated before we moved beyond the deadzone");
 
-            MixedRealityPlayspace.PerformTransformation(p => p.RotateAround(Vector3.zero, Vector3.up, 45));
-            yield return new WaitForFixedUpdate();
-            yield return null;
-
-            Assert.AreEqual(targetTransform.rotation, Quaternion.identity, "Target rotated before we moved beyond the deadzone");
```

Comment:
Do you have a little more context on why we removed this part of the test? i.e. I'm guessing this is due to the change to maxViewVerticalDegrees.

Or is the though here that since we already have deadzone detection in the test on the previous lines it's okay?

If that's the case, then that's fine - if not, we may want to change this by lowering the angle (i.e. 45->35 or something and then up the other rotation from 45->55 so that the same total rotation is occuring, but done in a way that is affected by the deadzone)

Reply:
@wiwei These lines were generating test failure even if updated default values are within the allowed range for the user's custom value through the input field. I thought before and after rotating from 0 to 90 degrees would be enough for this test. I just tested with 35 and 55 degrees and fails the test. It seems like I am misunderstanding something? : )